binaries/data/mods/public/gui/prelobby/common/terms/Terms_of_Use.txt

```diff
@@ -1,13 +1,15 @@
 [font="sans-bold-18"]0 A.D. Empires Ascendant Multiplayer Lobby Terms of Use[/font]
 
-[font="sans-bold-14"]Document Date:[/font] 2018-08-22
+[font="sans-bold-14"]Document Date:[/font] 2018-09-25
 
 [font="sans-bold-14"]By using the 0 A.D. Multiplayer Lobby, you agree to:[/font]
  1. Only create one account per unique user on the service unless authorized by Wildfire Games.
  2. Not post profane statements, rude humor, pornographic content, or discriminatory comments on the service.
  3. Not purposefully demean the worth of others using the service.
  4. Not use the service to promote specific goods, services, or products unless authorized by Wildfire Games.
  5. Not impersonate other users of the service and only use your own nickname.
- 6. Not send spam, irrelevant or inappropriate messages to a large number of recipients.
- 7. Not attempt to artificially adjust any user of the service's rating or any of the statistics which impact it. (Examples of this are, but are not limited to: cheating in ranked games, reverse engineering the service, and taking advantage of other users of the service.)
- 8. Allow yourself to be removed from the service if at any time a moderator determines your behavior is not consistent with these rules.
+ 6a. Not disclose other users' personal data (includes, but is not limited to: real name, identification number, location data, discriminating factors) apart from the pseudonym they use on the service.
+ 6b. Avoid disclosing personal information about yourself, especially if you are below 16 years of age.
```

Comment:
As said above. I split 6 into 6a and 6b because it's a single thing, but it was getting long, but it can be merged or become 7.